builder/builder-config/src/test/java/io/helidon/builder/config/ConfigBeanMapperHolderTest.java

```diff
@@ -29,7 +29,7 @@ class ConfigBeanMapperHolderTest {
     @Test
     void testGlobalInstanceIsEmpty() {
         assertThat(ConfigBeanMapperHolder.configBeanMapperFor(Object.class),
-                   is(optionalEmpty()));
+                                 is(OptionalMatcher.optionalEmpty()));
```

Comment:
This refactoring does not make any sense. Original was better.

Reply:
pushed